src/widgets/AppWindow.js

```diff
@@ -702,7 +717,6 @@ export default class AppWindow extends createjs.Container {
 
     // Implementing BUTTON
     const submitCallBack = () => {
-      // todo: this code results in a new createjs.Text for each click of the submit button.  This code needs to be fixed to either cleanup the previous createjs.Text instance or use the same instance for all clicks where the displayed string gets updated.
```

Comment:
I believe this todo comment is still valid as each button press will create a new createjs.Text object and add it to the canvas without cleaning up any previous ones.